model_api/cpp/install_dependencies.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+#Install OpenCV
+apt-get install libopencv-dev 
```

Comment:
You may want to put this after `apt update` to ensure the latest registry is used

Reply:
Good catch. Will do this.